client/src/components/Form/FormElement.vue

```diff
@@ -274,6 +275,11 @@ const isOptional = computed(() => !isRequired.value && attrs.value["optional"] !
                 v-model="currentValue"
                 :options="attrs.options"
                 :multiple="attrs.multiple" />
+            <FormDialog
+                v-else-if="props.type == 'data_dialog'"
```

Comment:
For consistency recommending to use `===` here like it has been done for the other input elements. Thank you for working on this.